### PR TITLE
Added lag column to the kafka plugin

### PR DESCRIPTION
**Pull request by ebarriosjr**
Signed-off-by: ebarrios <esteban.barrios@trivago.com>

## Hey, I just made a Pull Request!

Added lag column to the kafka main table.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4842-gmgwps6chumb8owopaih4ixyhgtxijya.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6042546c65f8177c48035d06

File: plugins/kafka/src/components/ConsumerGroupOffsets/ConsumerGroupOffsets.tsx [hunk 1]
```diff
@@ -57,6 +57,17 @@ const generatedColumns: TableColumn[] = [
...
+    title: 'Lag',
+    field: 'lag',
+    render: (row: Partial<TopicPartitionInfo>) => {
+      let lag = undefined;
+      if (row.topicOffset && row.groupOffset) {
+        lag = +row.topicOffset - +row.groupOffset;
+      }
+      return <>{lag ?? ''}</>;
+    },
+  },
```
Comment (OrkoHunter) [inline plugins/kafka/src/components/ConsumerGroupOffsets/ConsumerGroupOffsets.tsx:70]: Could we add test cases for this? :)

**Review by OrkoHunter — APPROVED** [review]

File: plugins/kafka/src/components/ConsumerGroupOffsets/ConsumerGroupOffsets.tsx (hunk 1 above)
Comment (ebarriosjr) [reply plugins/kafka/src/components/ConsumerGroupOffsets/ConsumerGroupOffsets.tsx:70]: Hi @OrkoHunter I added a test case. Please take a look and let me know what you think :)

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4842-gmgwps6chumb8owopaih4ixyhgtxijya.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6042546c65f8177c48035d06

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**